Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG VERSION=lts
+ARG VERSION=20.11.0
```

Comment:
Revert, lts is the default value for convenience but all workflows cat .nvmrc for repeatability